project/api/tests/stv_tests.py

```diff
@@ -168,3 +168,26 @@ def test_api_can_not_create_stv(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         response = self.client.post(url, data_overlapping)
         self.assertEqual(response.status_code, status.HTTP_409_CONFLICT)
+
+    @authorized
```

Comment:
Do we need an authorization to perform a get request for stv-history-list?

Reply:
After closer inspection yes we do, do you think an unauthorized user would have need to see historical records? If so we can change it

Reply:
For me it would be easier without extra headers on GET, because I already have all the coding done for this.
On the other side, I can't object against doing it as a protected with authorization method.